projects/ngx-sub-form/src/lib/ngx-sub-form.component.ts

```diff
@@ -47,6 +43,24 @@ export abstract class NgxSubFormComponent<ControlInterface, FormInterface = Cont
     }, 0);
   }
 
+  private mapControls(
+    controlTransformationCallback?: (ctrl: AbstractControl, key: string) => any,
```

Comment:
Better named `mapControl`

Reply:
Not sure why. It's to map all the controls

Reply:
I mean the first arg not the method

Reply:
Oh right. Solved.